modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm

```diff
@@ -179,7 +179,7 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/hands.dmi'
 	icon_state = "peacekeeper_gripper_gloves"
 
-//PEACEKEEPER BELTS
+//PEACEKEEPER BELTS + SHOULDER HOLSTER CHANGES
```

Comment:
We have a seperate override file for the new bluesec, this is all the outdated Peacekeeper stuff; it'd be better if this was in a more used file. Look for whichever one is called Goofsec (that really needs to be renamed)

Reply:
did that